src/pages/drawing/drawing.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div>
+    <h2>{{ title }}</h2>
+    <p>
+      To start drawing send <InlineCode>StartDrawingRequest</InlineCode>.
+      <RunExampleButton @click="startDrawing">DrawTools.StartDrawingRequest</RunExampleButton>
```

Comment:
Maybe change the label to something that suggests what the user should do or "Activate draw mode on map" and instruct the user to start clicking on the map or something?